operator/main.py

```diff
@@ -662,18 +665,16 @@ def handle_deleted(core_v1_client, obj):
 
     elif pv_count == 0:
 
-        if storage_info_data.get("type") == "External":
+        hostvol_type = storage_info_data.get("type")
+
+        if hostvol_type == "External":
             # We can't delete external volume but cleanup StorageClass and
             # Configmap
-            volname = "kadalu.external." + volname
-            lib_execute(KUBECTL_CMD, DELETE_CMD, "sc", volname)
-            logging.info(logf(
-                "Deleted Storage class",
-                volname=volname,
-            ))
+            delete_storage_class(volname, hostvol_type)
             delete_config_map(core_v1_client, obj)
 
         else:
+            delete_storage_class(volname, hostvol_type)
```

Comment:
nitpick: For simplicity, this line and `delete_config_map()` line can move out of the if/else logic as its same code in here.

Reply:
Done